utils/snappy-move-results/generate_metadata.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+MASTER_COUNT=`oc get nodes -l node-role.kubernetes.io/master | grep -v NAME | wc -l`
+WORKER_COUNT=`oc get nodes -l node-role.kubernetes.io/worker | grep -v NAME | wc -l`
+INFRA_COUNT=`oc get nodes -l node-role.kubernetes.io/infra | grep -v NAME | wc -l`
+
+master=`oc get nodes -l node-role.kubernetes.io/master | grep -v NAME -m 1 | awk '{print $1}'`
+worker=`oc get nodes -l node-role.kubernetes.io/worker | grep -v NAME -m 1 | awk '{print $1}'`
+infra=`oc get nodes -l node-role.kubernetes.io/infra | grep -v NAME -m 1 | awk '{print $1}'`
+
+MASTER_NODE_TYPE=`oc describe node $master | grep "node.kubernetes.io/instance-type" | grep -oP '=\K(.*)$'`
```

Comment:
Maybe use jsonpath instead of grep: https://kubernetes.io/docs/reference/kubectl/jsonpath/? Thoughts?

Reply:
will give it a try!